.github/workflows/push.yaml

```
@@ -205,94 +205,15 @@ jobs:
           pattern: macos-sparkle-update-*
           delete-merged: true
 
-  create-windows-update:
-    name: Create Windows Update 🥩
+  sign-windows-build:
+    name: Windows Signing ✍️
+    uses: obsproject/obs-studio/.github/workflows/sign-windows.yaml@744443d67fda3b40bd763206d7c072c9c1cb9981
```

```suggestion
    uses: .github/workflows/sign-windows.yaml@744443d67fda3b40bd763206d7c072c9c1cb9981
```

Shouldn't that be sufficient or does that also interfere with the hash check somehow?

---

Based on this it has to include the repo if you want to specify a commit hash: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_iduses

---

Got it, so we have to use it to ensure the correct hash is used.

---

It sounds like this is concern is resolved then?